apps/passport-server/src/services/telegramService.ts

```diff
@@ -53,6 +64,9 @@ export class TelegramService {
       "Research Workshop ZK Bot manages the Research Workshop Telegram group using ZKPs"
     );
 
+    const menu = new Menu("pcdpass-menu"); // should this be a class member?
```

Comment:
i think there's no need, unless we need to access this menu again in another class method

Reply:
Closed by 5c7e4e07